src/components/SettingsLoader.vue

```diff
@@ -1,18 +1,52 @@
 <template>
     <div id="settings-loader">
-        <slot v-if="isLoaded" />
+        <slot v-if="phase === PHASES.LOADED" />
 
-        <div v-else :class="['modal', 'z-top', {'is-active': error}]">
+        <div v-if="phase > PHASES.LOADED" class="modal z-top is-active">
```

Comment:
Relying on enum ordering without it being explicitly mentioned on the enum declaration seems potentially fragile/error prone if changed in the future.

Reply:
[Explicitly mentioning](https://github.com/ebkr/r2modmanPlus/pull/917/commits/8cacfe28b284832caa85493bd2a01f12b30d59cf#diff-449d18893f0dea4f8c02461069f6129fde06e2558467115d7766953e2f3b43abR67) it now.